stl/inc/algorithm

```diff
@@ -2847,6 +2847,40 @@ namespace ranges {
             _STL_INTERNAL_STATIC_ASSERT(sentinel_for<_Se2, _It2>);
             _STL_INTERNAL_STATIC_ASSERT(indirectly_swappable<_It1, _It2>);
 
+#if _USE_STD_VECTOR_ALGORITHMS
+            using _Elem1 = remove_reference_t<iter_reference_t<_It1>>;
+            using _Elem2 = remove_reference_t<iter_reference_t<_It2>>;
+            if constexpr (same_as<_Elem1, _Elem2> && _Is_trivially_swappable_v<_Elem1> && //
+                          contiguous_iterator<_It1> && _Sized_or_unreachable_sentinel_for<_Se1, _It1> && //
+                          contiguous_iterator<_It2> && _Sized_or_unreachable_sentinel_for<_Se2, _It2>) {

```

Comment:
We conventionally wrap _before_ rather than after binary operators:
```suggestion
            if constexpr (same_as<_Elem1, _Elem2> && _Is_trivially_swappable_v<_Elem1> //
                          && contiguous_iterator<_It1> && _Sized_or_unreachable_sentinel_for<_Se1, _It1> //
                          && contiguous_iterator<_It2> && _Sized_or_unreachable_sentinel_for<_Se2, _It2>) {
```